static/src/javascripts/projects/commercial/modules/dfp/fill-advert-slots.js

```diff
@@ -22,13 +22,13 @@ const fillAdvertSlots = (): Promise<void> => {
     // This module has the following strict dependencies. These dependencies must be
     // fulfilled before fillAdvertSlots can execute reliably. The bootstrap (commercial.js)
     // initiates these dependencies, to speed up the init process. Bootstrap also captures the module performance.
-    const dependencies: Promise<void>[] = Promise.all([
+    const dependencies: Promise<void>[] = [
```

Comment:
I did it this way round, rather than change the type, because the original intention it to ensure the array elements really are Promises. Otherwise `Promise.all` co-erces the elements.